validation-test/compiler_crashers_2_fixed/sr15497.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %swift-frontend -typecheck  %s
+
+public struct {
```

Comment:
Forgot the name here.

Reply:
Not including a name was part of the test case in the bug report, so it might have been crucial to trigger the crash before. So: This is intentional.

Reply:
Okay, you need to add `not`  before %swift-frontend then if you don't want to deal with error messages.